doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+  clustered using Samba's CTDB technology. The smb module requires a
+  cephadm enabled Ceph cluster and deploys container images provided by
+  the samba-container project. The Ceph dashboard can be used to configure
+  smb clusters and shares. A new cephfs-proxy daemon is automatically
+  deployed to improve the scalibilty and memory usage when connecting
+  Samba to CephFS.
+
+CephFS
+------
+
+* Directories may now be configured with case-insensitive or normalized
+  directory entry names. This is an inheritable configuration making
+  it apply to an entire directory tree. For more information, see
+  https://docs.ceph.com/en/latest/cephfs/charmap/
+* Modifying the FS setting variable "max_mds" when a cluster is
+  unhealthy now requires users to pass the confirmation flag
+  (--yes-i-really-mean-it). This has been added as a precaution to tell the
+  users that modifying "max_mds" may not help with troubleshooting or recovery
+  effort. Instead, it might further destabilize the cluster.
+* EOPNOTSUPP (Operation not supported) is now returned by the CephFS
+  fuse client for `fallocate` for the default case (i.e. mode == 0) since
+  CephFS does not support disk space reservation. The only flags supported are
+  `FALLOC_FL_KEEP_SIZE` and `FALLOC_FL_PUNCH_HOLE`.
+
+Dashboard
+---------
+
+* There is now added support for NVMe/TCP (gateway groups, multiple
+  namespaces), multi-cluster management, oAuth2 integration, and enhanced
+  RGW/SMB features including multi-site automation, tiering, policies,
+  lifecycles, notifications, and granular replication.
+
+MGR
+---
+
+* Users now have the ability to force-disable always-on modules and the
+  removal of the restful and zabbix modules (both deprecated since 2020).
+  Note that the dashboard module's richer and better-maintained RESTful
+  API can be used as an alternative to the restful module, and the
+  prometheus module can be used as an alternative monitoring solution for
+  zabbix.
+
+
+RADOS
+-----
+
+* Long expected performance optimizations (FastEC) have been added for EC pools,
+  including partial reads and partial writes.
+* A new implementation of the Erasure Coding I/O code provides substantial performance
+  improvements and some capacity improvements. The new code is designed to optimize
+  performance when using Erasure Coding with block storage (RBD) and file storage
+  (CephFS) but will have some benefits for object (RGW) storage, in particular when
+  using smaller sized objects. A new flag ``allow_ec_optimizations`` needs to be set
+  on each pool to switch to using the new code. Existing pools can be upgraded once
+  the OSD and MON daemons have been updated. There is no need to update the clients.
+* The default erasure code plugin has been switched from Jerasure to ISA-L.
+* BlueStore now has better compression and a new, faster WAL (write-ahead-log).
+* All components have been switched to the faster OMAP iteration interface.
```

Comment:
Earlier this year Zac assiduously looked up the original paper and noted that it consistently lowercases omap, so we've been favoring that in docs.  ymmv.